Activities/Database/UiPath.Database/UiPath.Database.csproj

```diff
@@ -22,6 +23,10 @@
     <PackageReference Include="UiPath.Workflow" />
     <PackageReference Include="System.Activities.Core.Presentation" />
   </ItemGroup>
+  <ItemGroup>
+    <PackageReference Include="System.Data.SqlClient" Version="4.8.2" />
+    <PackageReference Include="UiPath.Activities.Api" />
```

Comment:
Make it Private>All - this is just a development dependency. The robot will provide the actual dll at runtime

Reply:
made private, as requested